deploy/utils.sh

```diff
@@ -477,3 +478,35 @@ get_logs() {
     get_conjur_logs_container
     get_app_logs_container
 }
+
+# Return the pods information of the test-env without the headers
+# For example: 'test-app-5-fab52b20-0 secret-provider-0 1/1 Running 1 20m'
+get_pods_info() {
+  $cli_with_timeout get pods \
+    --namespace="$APP_NAMESPACE_NAME" \
+    --selector app=test-env \
+    --no-headers
+}
+
+# Return the pod name of a given namespace and app name
+# For example: 'secret-provider-0'
+get_pod_name() {
+  local namespace=$1
+  local selector=$2
+  local cli=${3-$cli_with_timeout}
+
+  pod_name=$(
+    $cli get pods \
+      --namespace=${namespace} \
+      --selector ${selector} \
+      -o jsonpath='{.items[].metadata.name}'
+  )
+
+  if [[ -z ${pod_name:-""} ]]; then
```

Comment:
```sh
  if [[ -z $pod_name ]]; then
```
should work here since `pod_name` will always be defined (line 498).

Nicely formatted bash, btw!

Reply:
this method was written by @izgeri :)